games/j.yaml

```
@@ -58,6 +58,24 @@
   url: https://bytonic.de/html/jake2.html
   info: Java port of the Quake2 game engine
 
+- name: Javascript BoulderDash
+  originals:
+  - Boulder Dash
+  type: clone
```

```suggestion
  type: remake
```
the game uses original game assets